src/mono/browser/browser.proj

```diff
@@ -339,7 +339,7 @@
       { "identity": "WasmSingleFileBundle",         "defaultValueInRuntimePack": "$(WasmSingleFileBundle)" },
       { "identity": "WasmEnableSIMD",               "defaultValueInRuntimePack": "$(WasmEnableSIMD)" },
       { "identity": "WasmEnableExceptionHandling",  "defaultValueInRuntimePack": "$(WasmEnableExceptionHandling)" },
-      { "identity": "WasmPerfTracing",              "defaultValueInRuntimePack": "$(WasmPerfTracing)" },
```

Comment:
This is documented switch (https://learn.microsoft.com/en-us/aspnet/core/blazor/performance/webassembly-event-pipe-diagnostics). Does this rename need to be tracked as a breaking change?

Reply:
> Does this rename need to be tracked as a breaking change?

No, all of this is new in Net10. 
That document was created for Net10 preview4, we just need to update the doc.
We may need to mention it in next preview6 release notes.